RPi4.md

```diff
@@ -0,0 +1,222 @@
+
+### Flash your SD card with Raspbian Linux
+
+Assuming  you’re running macOS, first you need to identify which device is your SD card. Plug your SD card into your SD card reader and type:
+
+```bash
+sudo -sdiskutil list
+```
+
+You’ll see a list of disks like this:
+
+![RPI4 Console](/img/RPI4Terminal1.png)
+
+Here’s where your computer might be different from the above and you need to be careful. From the above list of disks on my computer, I can identify `disk0` and `disk1` are my mac’s internal hard disks. But that 32GB “external” and “physical” `disk2` is the same size as my SD card and has some Windows partition on it, so I can identify my SD card is `disk2`.
+
+🚨 **WARNING: You need to correctly identify the drive number of your SD card, and  modify the following commands before typing them, or you could  accidentally erase your computer’s hard drive instead.** 🚨
+
+Don’t copy and paste the following, you need to replace the 3 instances of `diskX` in these commands with your actual drive ID. For me this was `disk2`,but it might be different for you. The following commands will erase the SD card, and then write the Raspbian image to the SD card, so be careful not to mess up these commands.
+
+```bash
+diskutil unmountDisk diskX
+dd if=/dev/zero of=/dev/rdiskX bs=4m count=100
+dd if=/path/to/raspbian.img of=/dev/rdiskX bs=4m
+```
+🚨 **WARNING:  If you incorrectly flash the wrong drive, you could erase your  computer’s hard drive instead of the SD card. Double check you have the  disk ID correct so you don’t accidentally erase your data.** 🚨
+
+If all went well, you should see the commands return something like this:
+
+![RPI4 Console](/img/RPI4Terminal2.png)
+
+Next, enable SSH at bootup so you can remotely login, and finally eject the SD card so you can move it to the Raspberry Pi. The new SD card’s boot  partition should automatically be mounted on `/Volumes/boot`
+
+```bash
+touch /Volumes/boot/ssh
+diskutil eject disk2
+```
+
+![RPI4 Console](/img/RPI4Terminal4.png)
+
+## Booting up the Raspberry Pi
+
+After inserting the SD card into the Raspberry Pi, go ahead and connect the power and ethernet, and optionally the display and keyboard if you have those. It should boot up and get an IP address using DHCP. You can try searching for it with `ping raspberrypi.local` on your desktop PC, but if that doesn’t work you will need to login to your router to find its IP address.
+
+The IP address that my Raspberry Pi got was 192.168.1.5 so I SSH’d to that
+
+```
+ssh 192.168.1.5 -l pi
+```
+
+The  default password for the “pi” user is “raspberry”. After SSH’ing in, the first thing I want to do is check the device’s CPU temperature to  make sure the cooling system are working correctly:
+
+```bash
+sudo -svcgencmd measure_temp
+```
+
+Next, let’s change the password for the “pi” user.
+
+```bash
+passwd pi
+```
+
+![](C:\Users\Pavle\Documents\GitHub\btcpayserver-doc\btcpayserver-doc\btcpayserver-doc\img\RPI4Terminal4.png)
```

Comment:
@pavlenex here a local image path slipped in, which results in a broken image. Would be nice to have it unbroken ;)

Reply:
Fixed, thanks for spotting that! 467093ba155042224623687e6bfad9e54c8a3e70